src/libs/ReportUtils.ts

```diff
@@ -3134,6 +3134,21 @@ function goBackToDetailsPage(report: OnyxEntry<Report>) {
     Navigation.goBack(ROUTES.REPORT_SETTINGS.getRoute(report?.reportID ?? ''));
 }
 
+/**
+ * Go back to the previous page from the edit private page of a given report
+ */
+function goBackFromPrivateNotes(report: OnyxEntry<Report>, session: OnyxEntry<Session>) {
+    if (isEmpty(report) || isEmpty(session) || !session.accountID) {
+        return;
+    }
+    const currentUserPrivateNote = report.privateNotes?.[session.accountID]?.note ?? '';
+    if (isEmpty(currentUserPrivateNote)) {
+        goBackToDetailsPage(report);
```

Comment:
@shubham1206agra I think this function is missing the selfDM case that appears in `navigateToDetailsPage`. So I think we can add this case in this function instead of modified a new version of `navigateToDetailsPage`. WDYT?

Reply:
What's wrong here. Can you detail? Thanks.